spec/features/authorizations/user_cannot_view_other_user_pages_spec.rb

```diff
@@ -2,35 +2,37 @@
 
 describe "users" do
 
-  before do
-    @category = Category.create!(name: "pizza")
-    item = @category.items.create!(name: "item1", description: "item1desc", price: 5000, image_file_name: "default")
-    @user = User.create!(user_attributes)
-
-    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
-    
-    #@cart = Cart.new({"#{item.id}" => 10})
-    #allow_any_instance_of(ApplicationController).to receive(:cart).and_return(@cart)
-    
-    #  how do I stub this out???
-     visit "/"
-     click_link_or_button('pizza')
-
-     10.times do
-       click_link_or_button('Add To Cart')
-     end
-
-     click_link_or_button("Items in Cart")
+
+  it "cannot view other user pages" do
+    category = Category.create!(name: "pizza")
+    category.items.create!(name: "item1", description: "item1desc", price: 5000, image_file_name: "default")
+    user = User.create!(user_attributes)
+
+    visit "/"
+    sign_in(user)
+
+    click_on('pizza')
+
+    10.times do
+      click_on('Add To Cart')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.